frontend/src/Issues.vue

```diff
@@ -0,0 +1,197 @@
+<script>
+import Bool from './Bool.vue'
+import Choice from './Choice.vue'
+
+export default {
+  name: 'Issues',
+  data () {
+    return {
+      state: 'loading',
+      filters: {
+        new_for_revision: null,
+        analyzer: null,
+        path: null
+      },
+      choices: {
+        new_for_revision: [
+          {
+            name: 'New for revision',
```

Comment:
Nit: `New in this revision`?

Reply:
Another nit: in the UI, it's a little strange that the menu on top of this is "Filter by issues"